content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -95,10 +95,10 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    ```bash
    kubectl version --client
    ```
-   Or use this for detailed view of version:
+   Or use this for detailed view of version: [Some More About Output Options](https://kubernetes.io/docs/reference/kubectl/overview/#output-options)
```

Comment:
[This PR as-is]
![image](https://user-images.githubusercontent.com/46767780/160281414-eb8c269e-d019-47c4-8d15-ce08331e5cd4.png)

[My suggestion]
```suggestion
   Or use this for detailed view of version ([available output options](/docs/reference/kubectl/#output-options)):
```
(Please note that I updated the link URL too.)

Reply:
So adding to all the 3 👍🏼